notification.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"os"
+	"strconv"
+
+	mmmodel "github.com/mattermost/mattermost-server/v5/model"
+	"github.com/pkg/errors"
+)
+
+func send(webhookURL string, payload mmmodel.CommandResponse) error {
+	marshalContent, _ := json.Marshal(payload)
+	var jsonStr = []byte(marshalContent)
+	req, err := http.NewRequest("POST", webhookURL, bytes.NewBuffer(jsonStr))
+	req.Header.Set("X-Custom-Header", "aws-sns")
+	req.Header.Set("Content-Type", "application/json")
+
+	client := &http.Client{}
+	resp, err := client.Do(req)
+	if err != nil {
+		return err
```

Comment:
nit: wrap the error. Not a big deal at all.

Reply:
fixed thanks :)